components/cice/bld/configure

```diff
@@ -518,7 +518,7 @@ if ($print>=2) { print "CPP definitions set by configure: \'$cfg_cppdefs\'$eol";
 #-----------------------------------------------------------------------------------------------
 
 my $fp_filename      = 'Filepath';             # name of output filepath file
-my $cpp_filename     = 'CICE_cppdefs';         # name of output file for cice's cppdefs 
```

Comment:
Is there a reason we use "CCSM_cppdefs" instead of something with E3SM or CIME?

Reply:
It's a legacy name. I just changed it to match the other components.

Reply:
OK, just curious